tools/golang-json-rpc-tests/.env.sample

```diff
@@ -0,0 +1,7 @@
+# The operator private keys start with "0x" and are in hexadecimal format
+# Your account ECDSA hex-encoded private key (Ex: 0xb46751179bc8aa9e129d34463e46cd...)
+OPERATOR_PRIVATE_KEY='OPERATOR_PRIVATE_KEY'
+
+# Sample transaction already existing in the network. It will be used to check whether some of the RPC methods inquiring
+# transactions data are working correctly.
+TRANSACTION_HASH=4b970a3c02af09d646d2bc099b94d95cd2c9b7c92d1bf42e65eee0258016a234
```

Comment:
Q: shouldn't this be empty?
Other than mainnet all transaction hashes are not long lived

Reply:
Yes, we create the transactions during tests. This environment variable is not required, so I’ve removed it.